### PR TITLE
[RHPAM-1387] Allow set Smart router location to external URL in RHPAM

### PR DESCRIPTION
**Pull request by msmagnanijr**
Signed-off-by: Mauricio Magnani <mmagnani@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: smartrouter/image.yaml [hunk 1]
```diff
@@ -73,6 +73,8 @@ envs:
       description: "KIE server controller token for bearer authentication (Sets the org.kie.server.controller.token system property)"
     - name: "LAUNCH_DIR"
       value: /opt/rhpam-smartrouter/launch
+    - name: "JBOSS_HOME"
+      value: "/opt/eap"
```
[the remark this answers is not in the document]
Comment (spolti) [reply smartrouter/image.yaml]: Why setting JBOSS_HOME?
is some of the new dependencies added using it?

[the remark this answers is not in the document]
Comment (errantepiphany) [reply smartrouter/image.yaml]: Forgot to remove it.

[the remark this answers is not in the document]
Comment (errantepiphany) [reply smartrouter/image.yaml]: @spolti Sorry, my previous comment (which I just deleted) had to do with a different PR, LOL.

[the remark this answers is not in the document]
Comment (errantepiphany) [reply smartrouter/image.yaml]: @spolti: @msmagnanijr is re-using a module where it assumes JBOSS_HOME is set. For smartrouter, it wasn't set.

**Comment by errantepiphany** [comment]
Closing without merging. Replaced by https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/299

**Pull request closed without merging**